content/data-transparency/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Data transparency
+---
+
+# Data transparency
+
+## Data collected by the PrestaShop project
+
+The PrestaShop project may collect some data from stores running PrestaShop 8 and over for analytics purposes only. 
+
+This data is extremely important for project members to understand how our ecosystem uses PrestaShop and make informed decisions.
+
+PrestaShop does not collect any personal information.
+
+### Data we collect
+
+* PrestaShop version
+* Main front office URL for the shop
+
+### Concerned versions
+
+PrestaShop 8 or over, running the [Distribution API client module](https://github.com/PrestaShop/ps_distributionapiclient/) v1.0.3 and above.
+
+### When and how the data is collected
+
+PrestaShop provides a free-of-charge service allowing stores to receive updates to the project's modules directly in the store's Back Office, through the _Distribution API client_ module. When this module is active, it will regularly ping our services to retrieve the latest available version for the project's modules. Unless you opt-out to data sharing, the aforementioned store information will be sent over to our services during this process.
+
+Note: The PrestaShop software version will still be sent even if data sharing is turned off as it necessary to provide the service. However, this information won't be traced back to the store's URL when data sharing is disabled.
+
+### How to opt-out from data sharing
+
+##### Option 1: Do not use _Distribution API client_
+
+* During install, uncheck _Distribution API client_ from the list of modules to install.
+* Alternatively, uninstall the module from your store.
+
+##### Option 2: Disable data sharing
+
+1. In your Back office, go to Module mananager > Distribution API client > Configure.
+2. Turn "Share this store's public URL with PrestaShop" to _off_.
```

Comment:
This will be correct if we merge https://github.com/PrestaShop/ps_distributionapiclient/pull/17

however if only https://github.com/PrestaShop/PrestaShop/issues/31627 is implemented then it would rather be

```
##### Option 2: Disable data sharing

Set the environment variable PS_URL_TRACKING to true. You can do using the SetEnv Apache directive in a .htaccess file for example.
```

Reply:
So, what's the final verdict? is it going to the module settings or as an env variable @eternoendless @matks @fabienvallon ?

Reply:
Why not both?

Reply:
that's a question to @fabienvallon